src/renderer/components/app.tsx

```diff
@@ -112,9 +116,14 @@ export class App extends React.Component {
         preload: true,
       })
     ]);
+
+    setTimeout(() => {
+      // This is here so that the rest of react can respond to AllowedResources loading
+      this.isLoading = false;
+    }, 2000);
   }
 
-  @observable startUrl = isAllowedResource(["events", "nodes", "pods"]) ? routes.clusterURL() : routes.workloadsURL();
+  @observable startUrl = isAllowedResources("events", "nodes", "pods") ? routes.clusterURL() : routes.workloadsURL();
```

Comment:
this is a bit problematic, since `<Redirect>` component rendered before allowed resources are resolved.

Reply:
Will fix this